### PR TITLE
added support of styles, captions and width of columns

### PR DESCRIPTION
**Pull request by vovka-s**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -116,12 +120,24 @@ exports.execute = function(config, callback) {
             return write(sheetFront, callback);
         },
         function(callback) {
+            return async.eachSeries(cols, function(col, callback) {
+                var colStyleIndex = col.styleIndex || 0;
+                var res = '<x:col min="' + cn + '" max="' + cn + '" width="' + col.width + '" customWidth="1" style="' + colStyleIndex + '"/>';
```
Comment (celsasser) [inline index.js]: Would it make sense to default the col.width in the event that it's not specified so that it does go to undefined.

**Comment by celsasser** [comment]
This guy fixes col width and dates which would be good features in master.

File: index.js (hunk 1 above)
Comment (vovka-s) [inline index.js]: fixed. Thanks!

**Comment by vovka-s** [comment]
added default column width
added style for date. Use styleIndex=2 and set your local date format on 4th line of style.xml:
`<numFmt numFmtId="2" formatCode="dd.mm.yyyy;@"/>`

**Comment by celsasser** [comment]
Looks good!

**Comment by elwerene** [comment]
I somehow didn't get notified about this pull request. sorry! I will check it out on tuesday

**Comment by elwerene** [comment]
Thanks and sorry for the long delay!

**Comment by elwerene** [comment]
Please add yourself to the list of contributors! (by pull request)

**Pull request merged**